OpenRA.Mods.Common/Orders/BeaconOrderGenerator.cs

```diff
@@ -30,7 +30,7 @@ protected override IEnumerable<Order> OrderInner(World world, CPos cell, int2 wo
 		protected override IEnumerable<IRenderable> RenderAnnotations(WorldRenderer wr, World world) { yield break; }
 		protected override string GetCursor(World world, CPos cell, int2 worldPixel, MouseInput mi)
 		{
-			return "ability";
+			return "ability"; // TODO: [CursorReference]
```

Comment:
I didn't follow the entire discussion, was this left for later intentionally?

Reply:
This one is not easily fixable because this PR is not set up to lint UI-defined cursors at all. This should be left to a followup PR.